models/role.js

```diff
@@ -0,0 +1,11 @@
+const { Schema, model } = require('mongoose')
+
+const roleSchema = new Schema({
+  value: {
+    type: String,
+    unique: true,
+    required: true
+  }
+})
+
+module.exports = model('Role', roleSchema)
```

Comment:
maybe will be better move models names to consts

Reply:
because you use this a few times

Reply:
following the pattern that was created before me, but discussed your suggestion and was told not to change it